aspnet-core/Chart/Appearance.md

```diff
@@ -24,7 +24,7 @@ The chart displays different series in different colors, by default. You can cus
 {% endhighlight %}
 
 
-![](Appearance_images/Appearance_img1.png)
+![Custom color palette in ASP.NET Core Chart](appearance_images/aspnetcore-chart-custom-color-palette.png)
```

Comment:
Have you changed the folder name also? check all places.

Reply:
Yes
![image](https://github.com/syncfusion-content/asp.net-core-docs/assets/95272306/ae4d510b-66de-4c25-8da6-046b030642fc)